Automattic/wp-calypso
# Github Deployments: Show workflow file contents for template repository

Related to #

## Proposed Changes
This PR displays the contents of the workflow file to the user based on the selected template when creating a repository from template. This happens if the selected template allows advance deployments.

**No workflow file**

![image](https://github.com/Automattic/wp-calypso/assets/26530524/297439e5-1bc6-4e16-b9c9-8d3cd2ce8b65)

**With a workflow file**

![image](https://github.com/Automattic/wp-calypso/assets/26530524/e5467d1a-dd51-4547-8b14-9d43d90e6272)

## Testing Instructions



* Apply D139605-code
* Go to create repository screen and select the `Wordpress Plugin Boilerplate Powered` template
* Verify workflow file contents are rendered

## Pre-merge Checklist
